package.json

```diff
@@ -50,7 +50,9 @@
         "eslint-plugin-prettier": "3.1.0",
         "husky": "3.0.2",
         "prettier": "1.18.2",
-        "pretty-quick": "1.11.1"
+        "pretty-quick": "1.11.1",
+        "rxjs": "6.5.5",
```

Comment:
@allanchau, why is this and `rxjs-compat` required?

Reply:
@smebberson I was trying to build a project and it was complaining that it couldn't find the module rxjs and rxjs-compat. It might be worth removing it and testing again though

Reply:
@allanchau thats old, it's not in the code at all? Perhaps try removing it and testing again as you suggest and report back. Thanks.

Reply:
@smebberson I've removed it and it seems to work fine 🤷